tests/nextflow.config

```diff
@@ -1,10 +1,47 @@
 /*
-========================================================================================
-    Nextflow config file for running tests
-========================================================================================
-*/
-/*
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-     Config to store CODON DB paths and names
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run ebi-metagenomics/biosiftr -profile test
+
+----------------------------------------------------------------------------------------
 */
+
+
+
+params {
+    input                    = "${projectDir}/tests/test_samplesheet.csv"
+    biome                    = "human-vaginal-v1-0"
+    decontamination_indexes  = "${projectDir}/tests/bwamem2/"
+    reference_dbs            = "${projectDir}/tests/reference_dbs/"
+    singularity_cachedir     = "/hps/nobackup/rdf/metagenomics/service-team/singularity-cache/"
+}
+
+process {
+    errorStrategy                = 'terminate'
+    maxRetries                   = 0
+    resourceLimits = [
+        cpus: 2,
+        memory: 6.GB,
+        time: 2.h
+    ]
+}
+
+executor {
+    name                     = "slurm"
+    queueSize                = 200
+    queueGlobalStatus        = true
+    submitRateLimit          = "10 sec"
+    pollInterval             = "10 sec"
+}
+
+singularity {
+    enabled                  = true
+    autoMounts               = true
+    cacheDir                 = params.singularity_cachedir
+}
```

Comment:
This belongs -> https://github.com/EBI-Metagenomics/pipelines-nextflow-configs

Reply:
I got confused, we don't run unit tests on the cluster we run those on our laptops and github actions (at least now). I would remove these from the repo, is is possible to run the nf-tests locally?